db/migrate/20200406122000_add_statistics_to_portfolio.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatisticsToPortfolio < ActiveRecord::Migration[5.2]
+  def change
+    add_column :portfolios, :statistics, :jsonb, :default => {}
+  end
```

Comment:
@gmcculloug Looks good. Do we have to have a migration to set these stats on all existing Portfolios and PortfolioItems

Reply:
That would be a good idea, I'll get that added.

Was thing maybe this column should be more general, like `extra_info` (or `summary_information`) and `statistics` would be a top-level hash key and would support other top-level keys without having to create new columns.